en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md

```diff
@@ -14,13 +14,15 @@ Simple Network Management Protocol (SNMP) is an Internet-standard protocol for
      -  **snmp4j-2.1.0.jar**
      -  **snmp4j-agent-2.0.6.jar**
   
-2.  Enable SNMP in the `ei.toml` file, stored in the `<MI_HOME>/conf/` file by
-    adding the following entry: 
-    
-     ```toml
-     [synapse_properties]
-     'synapse.snmp.enabled'=true
-     ``` 
+2.  Enable SNMP in the `deployment.toml` file, stored in the `<MI_HOME>/conf/` file by
```

Comment:
@sajinieKavindya, shall we rewrite this section like this. Please check the formatting when adding my steps.

2.  Enable SNMP.

       1.  Open the  `<MI_HOME>/conf/deployment.toml` file.
       2. Add the SNMP configuration as shown below. 
             
             Add the `synapse.snmp.community.name` entry to configure the synapse SNMP community name.

Reply:
Done